.claude/commands/tm/clear-subtasks/clear-all-subtasks.md

```
@@ -0,0 +1,93 @@
+Clear all subtasks from all tasks globally.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)